avae/vis.py

```diff
@@ -230,10 +274,33 @@ def loss_plot(epochs, train_loss, val_loss=None, p=None):
         )
     else:
         plt.title("Loss")
+
+    plt.yscale("log")
+    plt.ylabel("Loss", fontsize=16)
+    plt.xlabel("Epochs", fontsize=16)
+    plt.xticks(fontsize=16)
+    plt.yticks(fontsize=16)
+    plt.legend()
+
     plt.tight_layout()
     if not os.path.exists("plots"):
         os.mkdir("plots")
     plt.savefig("plots/loss.png")
+    plt.close()
+
+    # only training loss
+    for i, loss in enumerate(train_loss):
```

Comment:
Is this to have a separate loss for training data for clarity of display?

Reply:
Yes, this is something I was going to discuss with you. Do you think in a paper we only need to show the training loss or also validation loss ? It's just that the loss plot although excellent for us, do you think looks really busy for publication ?

Reply:
I agree that it looks busy.

Having said that - showing validation plot is important because it can show overfitting.

But I do agree it looks a bit busy so for a publication maybe we could show the full plot side by side with a TOTAL loss plot only? That makes it less busy and a bit more clear while still illustrating the overfitting situation. What do you think?

Reply:
I think then figure size should depend on the number of classes ?